src/routes/(app)/project/[id]/[promptId]/+page.server.ts

```diff
@@ -61,13 +55,15 @@ export async function load({ depends, locals: { supabase }, params }) {
 		instancesReq.data || [],
 		promptsRes.data[0]
 	);
+	const suggestions = getSuggestions(supabase, promptsRes.data[0], false);
```

Comment:
what's the advantage of loading this here over something like `onMount`? Seems like this could delay page load quite significantly if we're querying openAI.

Reply:
This actually gets streamed to the frontend since it returns a promise so it doesn't block! 

There are various benefits to doing data fetching in +page, one less fetch request roundtrip, streaming, etc.

Reply:
https://kit.svelte.dev/docs/load#streaming-with-promise

Reply:
ah true, missed that, sgtm